modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -87,8 +86,11 @@ protected Integer getLivenessCheckPort() {
     @Override
     protected void configure() {
 
-        checkState(desiredCapabilities != null);
-        if (! customImageNameIsSet) {
+        if(desiredCapabilities == null) {
+            throw new IllegalStateException();
```

Comment:
Please take a look at description, this change is non-functional and comes from #390